DIFFERENCES.md

```diff
@@ -17,3 +17,9 @@
   Go version will handle this as "API error" while ruby version will fail to
   parse the response and handle this as "JSON error".
 - The Go HTTP client tries to reuse connections with keep-alive.
+- Docs for GET APIs call for URL-encoded query params. Ruby version sends a
+  JSON query in body (like for other verbs). Go implementation follows docs.
+- The connect.synchronizeProducts() returns deserialized slice of Product
```

Comment:
connect.syncProducts() - but not sure that this is a difference we need to document. The function is not exported.

Reply:
I'd rather keep it for developers as a sign of by-design change in behavior.